Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -123,13 +110,15 @@ CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedStrin
     }
 
     // Get the default locale for this user.
-    std::wstring localeName = _GetUserDefaultLocaleName();
+    // If the default locale is returned, find that locale name, otherwise use "en-us".
+    wchar_t localeNameBuf[LOCALE_NAME_MAX_LENGTH];
+    int defaultLocaleSuccess = GetUserDefaultLocaleName(localeNameBuf, LOCALE_NAME_MAX_LENGTH);
+    const wchar_t* localeName = defaultLocaleSuccess ? localeNameBuf : c_defaultUserLanguage;
 
```

Comment:
it _was_ a private method. i unrolled it because it would be clunky without the std::wstring return (caller would need to declare buffer/know size required, private function would only end up being two lines anyway, not saving much).

---
In reply to: [92735069](https://github.com/Microsoft/WinObjC/pull/1558#discussion_r92735069) [](ancestors = 92735069)